packages/mongo-nav/src/user-menu/UserMenu.spec.tsx

```diff
@@ -78,6 +79,22 @@ describe('packages/mongo-nav/user-menu', () => {
     expect(universityItem).toBeVisible();
   });
 
+  test('when "hasLegacy2fa" is set to false, does not render mfa menu item', () => {
```

Comment:
Can we add a test to make sure we're correctly showing/hiding the banner as well please

Reply:
See comment below about the banner not being a conditional render - I can add a test to ensure the banner is always rendered though

Reply:
Thank you!